configure.ac

```diff
@@ -787,6 +787,25 @@ else
 fi
 AM_CONDITIONAL(WITH_OFONO, test "${with_ofono}" = "yes")
 
+# DHCP client support with dhcpcanon
+AC_ARG_WITH([dhcpcanon],
+            AS_HELP_STRING([--with-dhcpcanon=yes|no|path], [Enable dhcpcanon support]))
+if test "$with_dhcpcanon" != "no"; then
```

Comment:
Could you also add `--with-dhcpcanon=no` to ` contrib/fedora/rpm/NetworkManager.spec`? After `--with-dhcpcd=no`